docs/NMOS With H.264.md

```diff
@@ -104,20 +101,18 @@ These attributes provide information for Controllers and Users to evaluate strea
 
   The level strings in this specification are included in the [NMOS Parameter Registers][]. Additional strings may be added there in the future.
 
-The Flow's `profile` and `level` attributes map to the `profile-level-id` parameter of the SDP transport file. See section SDP format-specific parameters.
+The Flow's `profile` and `level` attributes map to the `profile-level-id` parameter of the SDP transport file. See the [SDP format-specific parameters](#sdp-format-specific-parameters) section.
 
 The Flow's `profile` and `level` attributes map to the members profile_idc, level_idc and constraint_set\<N\>_flag of the AVC_video_descriptor of an MPEG2-TS transport stream. See section Multiplexed Flows.
```

Comment:
@alabou Which section should "Multiplexed Flows" refer to?

Reply:
I would say that the section would be "RTP transport based on RFC 2250" ... not "Multiplexed Flows". The required behavior of the Sender is described form the Receiver point of view ...